{
 "ETag": "W/\"6b35d5cdedf8c1180fce113cbcc43ed14fa99ca4777a80115ded6cd99ec804dc\"",
 "Last-Modified": "Fri, 15 Jul 2022 15:03:58 GMT",
 "base": {
  "repo": {
   "name": "google-cloud-error-reporting-feedstock"
  }
 },
 "closed_at": "2022-07-15T15:03:55Z",
 "created_at": "2022-07-14T17:20:32Z",
 "draft": false,
 "head": {
  "ref": "this_is_not_a_branch"
 },
 "html_url": "https://github.com/conda-forge/google-cloud-error-reporting-feedstock/pull/22",
 "id": 996934072,
 "labels": [
  {
   "color": "0e8a16",
   "default": false,
   "description": "Merge the PR when CI passes",
   "id": 2024420799,
   "name": "automerge",
   "node_id": "MDU6TGFiZWwyMDI0NDIwNzk5",
   "url": "https://api.github.com/repos/conda-forge/google-cloud-error-reporting-feedstock/labels/automerge"
  }
 ],
 "mergeable": null,
 "mergeable_state": "unknown",
 "merged": true,
 "merged_at": "2022-07-15T15:03:55Z",
 "number": 22,
 "state": "closed",
 "updated_at": "2022-07-15T15:03:58Z"
}